src/components/TipInput.vue

```diff
@@ -13,14 +13,22 @@
           v-if="!tipUrlStats.tokenTotalAmount.length || +tipUrlStats.totalAmountAe !== 0"
           :amount="tipUrlStats.totalAmountAe"
         />
+      </template>
```

Comment:
Actually, this template contains only one child node. Please, remove it by applying `v-if` to AeAmountFiat node.